src/vsr/superblock_quorums_fuzz.zig

```diff
@@ -109,11 +109,7 @@ fn test_quorums_working(
     const misdirect = random.boolean(); // true:cluster false:replica
     var quorums: Quorums = undefined;
     var headers: [4]SuperBlockHeader = undefined;
-    // TODO(Zig): Ideally this would be a [6]?u128 and the access would be
-    // "checksums[i] orelse random.int(u128)", but that currently causes the compiler to segfault
-    // during code generation.
-    var checksums: [6]u128 = undefined;
-    for (&checksums) |*c| c.* = random.int(u128);
+    var checksums: [6]?u128 = undefined;
```

Comment:
It's actually wrong!! It should be:

```diff
- var checksums: [6]?u128 = undefined;
+ var checksums: [6]?u128 = [_]?u128{null} ** 6;
```

However it requires changing the access to:

```diff
- .parent = checksums[copies[i].sequence - 1] orelse random.int(u128)
+ .parent = checksums[copies[i].sequence - 1] orelse parent: {
+    const parent = random.int(u128);
+    checksums[copies[i].sequence - 1] = parent;
+    break :parent parent;
+ },
```

Not sure if it has any advantage over the previous version!
WDYT about reverting the code just dropping the TODO message?

Reply:
sgtm!